package.json

```diff
@@ -29,7 +29,7 @@
     "grunt-contrib-imagemin": "^3.1.0",
     "grunt-contrib-jshint": "^1.1.0",
     "grunt-contrib-symlink": "^1.0.0",
-    "grunt-contrib-uglify": "^3.0.1",
+    "grunt-contrib-uglify": "^5.0.0",
```

Comment:
as this is unrelated to PR, on purpose?

Reply:
the version of uglify that we are using does not support maps. I am using one and it would make the build fail on the uglify step. I do not see any drawback from upgrading it